Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1717,7 +1730,33 @@ public JSONObject getFirstReferringParams() {
      */
     public JSONObject getLatestReferringParams() {
         String storedParam = prefHelper_.getSessionParams();
-        return convertParamsStringToDictionary(storedParam);
+        JSONObject latestParams = convertParamsStringToDictionary(storedParam);
```

Comment:
Let's put a loud warning here that says. "You're currently in deep link debug mode. Please comment out 'setDeepLinkDebugMode' to receive the deep link parameters from a real Branch link". Please put this in the above function as well.